app/auth/controllers.py

```diff
@@ -56,7 +56,8 @@ def callback_handling():
         user = User().create_or_update_user_from_callback(user_info.data)
         jwt_helper = JWT(app.config['API_KEY'], user.id)
         session.pop('github_token', None)
-        resp = make_response(render_template("dashboard.html", user=user,
+        g.current_user = user
```

Comment:
Why are you setting g.current_user here? Surely you have just set the cookie so on the next page request they will be logged in fine.